app/examples/vimium

```diff
@@ -1,39 +1,30 @@
-map <Esc> <action.toNormalMode>
-map <C-[> <action.toNormalMode>
+" Options
+set adblocker=update
+set downloadmethod=confirm
+set mintabwidth=250
+set norestoretabs
+set redirecttohttp
+set suspendonrestore=regular
+set tabreopenposition=previous
+set windowtitle=title
+set search=https://google.com/search?q=
+set searchwords=w~https://www.wikipedia.org/w/index.php?title=Special:Search&search=%s
+set ignorecase
```

Comment:
Unlike Vim, `ignorecase` and `incsearch` are actually true by default in Vieb, so we can also leave those out. Otherwise this is looking great!